backend/src/email/services/email-transactional.service.ts

```diff
@@ -35,16 +43,18 @@ async function sendMessage({
   const sanitizedSubject =
     EmailTemplateService.client.replaceNewLinesAndSanitize(subject)
   const sanitizedBody = EmailTemplateService.client.filterXSS(body)
+  // this only triggers if the subject or body is empty after sanitization
+  // highly unlikely in practice
   if (!sanitizedSubject || !sanitizedBody) {
     void EmailMessageTransactional.update(
       {
-        errorCode: 'Error 400: Message contains invalid HTML tags',
+        errorCode: EMPTY_MESSAGE_ERROR_CODE,
       },
       {
         where: { id: emailMessageTransactionalId },
       }
     )
-    throw new InvalidMessageError()
+    throw new EmptyMessageError()
```

Comment:
refactored this for greater precision

Reply:
See [above](https://github.com/opengovsg/postmangovsg/pull/1846/files#r1005439545)